### DISCUSSION TITLE
ts 类型检测错误后，怎么实现停止编译

### DISCUSSION THREAD

**Discussion by Lands-1203** [umijs/umi#9585] [Q&A]
最新版的umi中 怎么实现ts类型检测错误后停止编译，在tsconfig中配置了也不行
![image](https://user-images.githubusercontent.com/61341868/196912491-7c2efef9-8ef9-4a27-9622-971268cde74b.png)

**Comment by fz6m** [comment]
```ts
  forkTSChecker: {}
```

 - https://umijs.org/docs/api/config#forktschecker
